specification/cost-management/resource-manager/readme.go.md

```diff
@@ -49,3 +50,11 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'package-preview-2019-03' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-03-01/$(namespace)
```

Comment:
Technically this should be changed too, but since this package has already been released, we just leave this here unchanged.